scss/v2/components/tile.scss

```diff
@@ -0,0 +1,160 @@
+.tile {
+  font-size: 18px;
+  line-height: 1.78;
+  color: $grey2;
+  overflow: hidden;
+  box-shadow: 0 2px 0 0 rgba($grey1, .3);
+  background-color: $grey5;
+  padding-left: 16px;
+  padding-right: 16px;
+  padding-bottom: 16px;
```

Comment:
A stylistic thing more than anything, but seeing as we do it with a few `margin` properties below, perhaps we can format the `padding` here similarly?